### ISSUE TITLE
Pktvisor: "error":"500 invalid schema"

### ISSUE THREAD

**Issue by manrodrigues** [ns1labs/orb#747]
Agent container logs: After an error (for example, application of policy with repeated name) occurs, all other application generate ""error":"500 invalid schema", even the correct ones.

Some logs (on file) for steps:

- 2 policies applied
- 1 policy successfully removed
- another policy applied after 1 hour
- 1  dataset successfully removed
- Try to apply 1 policy with same name as one unapplied earlier (error)
- Try to apply edited policy (error)
- Try to apply new policy using new dataset both with not used names (error)
<br>

[docker_logs.log](https://github.com/ns1labs/orb/files/7849361/docker_logs.log)

**Comment by manrodrigues** [comment]
Another way to shutdown pktvisor and get the same error 500:

- apply a policy to an agent
- delete the policy
- delete the dataset to which the policy was linked
- check container logs


```
{"level":"warn","ts":1642432603.482642,"caller":"policyMgr/manager.go:129","msg":"failed to retrieve policy data","policy_id":"40e18d64-2549-4fdb-b2f4-de945dc779bc","error":"unknown policy ID"}
{"level":"info","ts":1642432659.5667155,"caller":"pktvisor/pktvisor.go:253","msg":"pktvisor stdout","log":"[2022-01-17 15:17:39.566] [visor] [info] REQUEST: GET /api/v1/policies/__all/metrics/bucket/1 200"}
{"level":"info","ts":1642432659.5670013,"caller":"pktvisor/pktvisor.go:334","msg":"scraped metrics for policy","policy":"policy2","policy_id":"b59494ef-4b68-492b-a905-cf5324940a64","payload_size_b":234}
{"level":"info","ts":1642432659.5671575,"caller":"pktvisor/pktvisor.go:334","msg":"scraped metrics for policy","policy":"policy3","policy_id":"7f1d044d-f901-4eb7-bffb-2aafa4dea2bd","payload_size_b":234}
{"level":"info","ts":1642432659.9248722,"caller":"pktvisor/pktvisor.go:352","msg":"scraped and published metrics","topic":"channels/2cf4377a-fd75-42d6-af9f-320c82f04d39/messages/be/pktvisor/m/f","payload_size_b":468,"batch_count":2}
{"level":"info","ts":1642432683.167067,"caller":"policyMgr/manager.go:50","msg":"managing agent policy from core","action":"manage","name":"policy5","dataset":"35dcfbfe-296f-4435-8b14-5f80240e4c7b","backend":"pktvisor","id":"749e2ff1-1f07-4037-815a-e235b143632c","version":0}
{"level":"info","ts":1642432683.1702805,"caller":"pktvisor/pktvisor.go:253","msg":"pktvisor stdout","log":"[2022-01-17 15:18:03.170] [visor] [info] policy [policy5]: parsing"}
{"level":"info","ts":1642432683.1707473,"caller":"pktvisor/pktvisor.go:253","msg":"pktvisor stdout","log":"[2022-01-17 15:18:03.170] [visor] [info] policy [policy5]: instantiating Tap: default_pcap"}
{"level":"info","ts":1642432683.1714432,"caller":"pktvisor/pktvisor.go:253","msg":"pktvisor stdout","log":"[2022-01-17 15:18:03.171] [visor] [info] policy [policy5]: instantiating Handler handler_net_1 of type net"}
{"level":"info","ts":1642432688.1725478,"caller":"pktvisor/pktvisor.go:253","msg":"pktvisor stdout","log":"[2022-01-17 15:18:08.172] [visor] [info] REQUEST: POST /api/v1/policies 500"}
{"level":"warn","ts":1642432688.1725516,"caller":"pktvisor/pktvisor.go:185","msg":"yaml policy application failure","policy_id":"749e2ff1-1f07-4037-815a-e235b143632c","policy":"version: \"1.0\"\nvisor:\n    policies:\n        policy5:\n            handlers:\n                modules:\n                    handler_net_1:\n                        config: {}\n                        filter: {}\n                        type: net\n            input:\n                input_type: pcap\n                tap: default_pcap\n            kind: collection\n"}
{"level":"info","ts":1642432688.1726334,"caller":"pktvisor/pktvisor.go:253","msg":"pktvisor stdout","log":"[2022-01-17 15:18:08.172] [visor] [error] {\"error\":\"invalid schema\"}"}
{"level":"warn","ts":1642432688.172654,"caller":"policyMgr/manager.go:154","msg":"policy failed to apply","policy_id":"749e2ff1-1f07-4037-815a-e235b143632c","policy_name":"policy5","error":"500 invalid schema"}
{"level":"info","ts":1642432719.5669537,"caller":"pktvisor/pktvisor.go:253","msg":"pktvisor stdout","log":"[2022-01-17 15:18:39.566] [visor] [info] REQUEST: GET /api/v1/policies/__all/metrics/bucket/1 200"}
{"level":"info","ts":1642432719.567088,"caller":"pktvisor/pktvisor.go:334","msg":"scraped metrics for policy","policy":"policy2","policy_id":"b59494ef-4b68-492b-a905-cf5324940a64","payload_size_b":234}
{"level":"info","ts":1642432719.5672185,"caller":"pktvisor/pktvisor.go:334","msg":"scraped metrics for policy","policy":"policy3","policy_id":"7f1d044d-f901-4eb7-bffb-2aafa4dea2bd","payload_size_b":234}
{"level":"info","ts":1642432719.724215,"caller":"pktvisor/pktvisor.go:352","msg":"scraped and published metrics","topic":"channels/2cf4377a-fd75-42d6-af9f-320c82f04d39/messages/be/pktvisor/m/f","payload_size_b":468,"batch_count":2}
{"level":"info","ts":1642432779.5672636,"caller":"pktvisor/pktvisor.go:253","msg":"pktvisor stdout","log":"[2022-01-17 15:19:39.567] [visor] [info] REQUEST: GET /api/v1/policies/__all/metrics/bucket/1 200"}
{"level":"info","ts":1642432779.5673616,"caller":"pktvisor/pktvisor.go:334","msg":"scraped metrics for policy","policy":"policy2","policy_id":"b59494ef-4b68-492b-a905-cf5324940a64","payload_size_b":234}
{"level":"info","ts":1642432779.567448,"caller":"pktvisor/pktvisor.go:334","msg":"scraped metrics for policy","policy":"policy3","policy_id":"7f1d044d-f901-4eb7-bffb-2aafa4dea2bd","payload_size_b":234}
{"level":"info","ts":1642432779.9283497,"caller":"pktvisor/pktvisor.go:352","msg":"scraped and published metrics","topic":"channels/2cf4377a-fd75-42d6-af9f-320c82f04d39/messages/be/pktvisor/m/f","payload_size_b":468,"batch_count":2}
```

**Comment by manrodrigues** [comment]
Fixed. Full logs:

[logs.txt](https://github.com/ns1labs/orb/files/7933860/logs.txt)


Last commit: db912ac
